pkg/blobinfocache/boltdb/boltdb.go

```diff
@@ -22,6 +23,8 @@ var (
 
 	// uncompressedDigestBucket stores a mapping from any digest to an uncompressed digest.
 	uncompressedDigestBucket = []byte("uncompressedDigest")
+	// digestCompressorBucket stores a mapping from any digest to a compressor, or blobinfocache.Uncompressed
```

Comment:
Adding `It may not exist in caches created by older versions, even if uncompressedDigestBucket is present.`  Let me know if you prefer your suggested text instead.